packages/rancher-supportability-review/charts/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+annotations:
+  catalog.cattle.io/auto-install: rancher-supportability-review-crd=match
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Supportability Review
+  catalog.cattle.io/namespace: sr-operator-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/permits-os: linux
+  catalog.cattle.io/provides-gvr: sr.cattle.io.reviewbundles/v1
+  catalog.cattle.io/rancher-version: '>= 2.11.0 < 2.12.0'
+  catalog.cattle.io/release-name: rancher-supportability-review
```

Comment:
Should set `catalog.cattle.io/kube-version: '>= 1.30.0-0 < 1.33.0-0'` somewhere to signify which Kubernetes versions this is valid for. These correspond 1-1 to with the supported versions by the rancher line

Reply:
Thank you for comment. Fixed and rebased.